Backbase/variants
# Add support for Environment Variables with GYB

### What does this PR do
- Creates 'Variants.Secrets' in 'Variants.swift' containing static variables for every custom property whose destination is 'project' and reads from environment variables
- These values are encrypted with a xor cipher using a salt that’s generated randomly each time.

### How can it be tested
- Adhere to the [latest changes](https://github.com/Backbase/variants/commit/2bad20dc1ab9ac6275daf6db6e1ba475931063bb#diff-88e254e2c28df11e815768543a21790e7d60b64b331abcf1d324ef6f768e4043) in `variants.yml`
- Include a custom property whose value reads from environment variable: `value: "{{ envVars.MY_ENVIRONMENT_VAR }}"`
- Switch to this respective variant `variants switch --variant A_VARIANT`
- `Variants.swift` should contain struct `Secrets` with your encrypted static var.

### Task
resolves #87 

### Checklist:
